packages/components/src/Chip/Chip.tsx

```diff
@@ -16,11 +16,12 @@ export const Chip = ({
   invalid,
   label,
   value,
+  dataTestID,
```

Comment:
The jsdoc comment mentions it defaults to `chip-wrapper` but that doesn't seem to be the case
https://github.com/GetJobber/atlantis/pull/1995/files#diff-93fff87012a73ee50578dd51a34c028ffeea01f74ada82dc08cf121ef8b80011R10

I may also be worth having it follow our other testID formats such as `ATL-CHIP-WRAPPER`

Reply:
I noticed it is defaulted in `Chips` usages but since `Chip` is exported separately it should be defaulted here

Reply:
ah yeah that default isn't true. I'm not sure if it's necessary to have a default either.

we might be fine keeping it optional, and then for our specific usage in `InternalChip` we pass the `ATL-Chip-Wrapper` or whatever we think makes sense